### ISSUE TITLE
Decoding BUG, grey frame for an 8-bit HEVC elementary stream

### ISSUE THREAD

**Issue by superbin** [GPUOpen-LibrariesAndSDKs/AMF#154]
[grey_frame_hevc_8b.h265](https://1drv.ms/u/s!AuOUOJzgRhF8ihRuZPbaSqJP6XvS)
[grey_frame.PNG](https://1drv.ms/u/s!AuOUOJzgRhF8ihXf__19787JLNli)
Graphics Card: Radeon RX 580 Series
Driver Version: 25.20.15003.5010/Win10 64
Reproduce step:
1. In amf SimpleDecoder project, set the input file to grey_frame_hevc_8b.h265.
2. Run.
3. Play the result nv12 file with a rawplayer, got grey frame.

PS. Play grey_frame_hevc_8b.h265 using other player like vlc is OK.

**Comment by superbin** [comment]
The actual height is 2160, but this file has 

- pic_height_in_luma_samples= 2176
- conformance_window_flag = 1
- conf_win_left_offset = 0
- conf_win_right_offset = 0
- conf_win_top_offset = 0
- conf_win_bottom_offset = 8

Does AMF support this type files?

**Comment by MikhailAMD** [comment]
I checked this clip. Here is the problem: The actual decoding height is 2176 but SPS NAL Unit defines cropping that makes the clip height as 2160. 
Normally SPS cropping is used as this: let's assume we want a clip height as 2168. This means that column of macroblocks (16x16 pixels) should be 136 to cover whole area and decoding height would be 136*16 = 2176. But this is larger then the desired clip height. In this case SPS cropping will indicate subtraction of 8 pixels to make the expected size 2168. 
In your case SPS bottom cropping should be 0 but it is set as 16.
I tried to parse the stream with AMF sample parser and with FFmpeg version we use in AMF using PlaybackHW sample. They both define decoder height as 2160 and decoding is wrong because of that.
Vega analyzer recognises this clip as the valid one. Bit really, this is not a bug but weird use of SPS parameters.
